### DISCUSSION TITLE
D-Link DWA-181 Dual Band AC1300 Usb 2.0 Working on Big Sur 11.7

### DISCUSSION THREAD

**Discussion by alsvieira** [chris1111/Wireless-USB-Big-Sur-Adapter#225] [Show and tell]
Just to give feedback that D-Link DWA-181 Dual Band AC1300 Usb 2.0 is working with your driver on BigSur 11.7 and connect to a Inateck 4 Ports PCI-E to USB 3.0 on a Mac Pro 4,1 upgraded to 5,1.  My TP-Link TL-WDN4800 N900 stop working with BigSur...tried different setup's on opencore but no luck with wifi. This usb wifi and the driver solve the problem. Great/full  speed also with the connection. Thank you for your great work.

Best Regards

**Comment by chris1111** [comment]
Great thanks
i'm glad to hear it

**Comment by robsmuir** [comment]
After the latest MacOS update to Big Sur 11.7.7, my MacBook Pro eventually crashed. On reboot, the tp-link AC1300 Nano stopped working. 

I eventually disabled SIP again, and reinstalled the drivers. Partway through the process, I noticed that the WiFi adaptor appeared to start working again, so the reinstall may not have been necessary. It's all good now.

Your mileage may vary…
